src/components/NewCommentForm.tsx

```diff
@@ -75,25 +175,38 @@
             id="comment-body"
             name="body"
             placeholder="Type comment here"
-            className="textarea is-danger"
+            className={classNames('textarea', { 'is-danger': hasEmailError })}
```

Comment:
The error class for the comment text area should use `hasTextError` instead of `hasEmailError`. This will ensure that the correct error state is applied to the text area.